dev/df_new_loader.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
```

Comment:
Very well documented and useful. I am wondering if we should not move these lines to a proper testing function!

Reply:
Yes absolutely! The purpose of this script was to test during development. I'll open another PR to include a proper testing function.